labs-ios-starter/Model Controller/ProfileController.swift

```diff
@@ -14,13 +14,13 @@ class ProfileController {
     static let shared = ProfileController()
     
     let oktaAuth = OktaAuth(baseURL: URL(string: "https://auth.lambdalabs.dev/")!,
-                            clientID: "0oalwkxvqtKeHBmLI4x6",
-                            redirectURI: "labs://scaffolding/implicit/callback")
+                            clientID: "0oa18is3355KlyP5C4x7",
+                            redirectURI: "labs://cityspire/implicit/callback")
     
     private(set) var authenticatedUserProfile: Profile?
     private(set) var profiles: [Profile] = []
     
-    private let baseURL = URL(string: "https://labs-api-starter.herokuapp.com/")!
+    private let baseURL = URL(string: "https://cityspire-a-be.herokuapp.com/")! // https://labs-api-starter.herokuapp.com/
```

Comment:
You can get rid of the comment here, that URL isn't useful

Reply:
Oh, yeah. Sorry.